scripts/create_kind_cluster.sh

```diff
@@ -2,6 +2,11 @@
 # sample from https://kind.sigs.k8s.io/docs/user/local-registry/
 set -o errexit
 
+if [[ "${USE_KIND:-}" = "false" ]]; then
```

Comment:
I like the idea behind this change. It makes sense that we don't always use kind. We can change this script even further, make it generic and also change the name accordingly (something without `kind`).

Alternatively we can create a separate script for other types and add similar branching in the `Makefile`. This way we won't need to skip `kind` cluster creation inside of `create_kind_cluster.sh` script.

Reply:
I think in the future we can make something more flexible! For now I think it makes sense to de-couple the creation of a kind cluster with the operator deployment.